tabular/src/autogluon/tabular/models/fastainn/tabular_nn_fastai.py

```diff
@@ -529,6 +530,12 @@ def _estimate_memory_usage(self, X, **kwargs):
     def _get_hpo_backend(self):
         """Choose which backend(Ray or Custom) to use for hpo"""
         return RAY_BACKEND
+    
+    def _get_maximum_resources(self) -> Dict[str, float]:
+        # fastai model trains slower when utilizing virtual cores and this issue scale up when the number of cpu cores increases
```

Comment:
Can we tie this to a GitHub issue?

Reply:
ditto for torch

Reply:
https://github.com/autogluon/autogluon/issues/3146

Reply:
Please include github issue link in-line